saos-restful-web-api/src/main/java/pl/edu/icm/saos/api/formatter/LawJournalEntryCodeFormat.java

```diff
@@ -0,0 +1,19 @@
+package pl.edu.icm.saos.api.formatter;
```

Comment:
`LawJournalEntryCodeFormat` is slightly binded with api project.
To this annotation to work it is needed `LawJournalEntryCodeFormatterFactory` but implementation of it in case of invalid input  throws `WrongRequestParameterException` which is specific to api project.